src/deploy/functions/release.js

```diff
@@ -152,8 +152,47 @@ module.exports = function(context, options, payload) {
   var deleteReleaseNames;
   var existingScheduledFunctions;
 
+  // Collect all the functions that have a retry policy
+  var failurePolicyFunctions = functionsInfo.filter((fn) => {
+    return !!fn.failurePolicy;
+  });
+
+  var failurePolicyFunctionLabels = failurePolicyFunctions.map((fn) => {
+    return helper.getFunctionLabel(_.get(fn, "name"));
+  });
+  var retryMessage =
+    "The following functions will be retried in case of failure: " +
+    clc.bold(failurePolicyFunctionLabels.join(", ")) +
+    ". " +
+    "Retried executions are billed as any other execution, and functions are retried repeatedly until they either successfully execute or the maximum retry period has elapsed, which can be multiple days. " +
```

Comment:
I believe the maximum retry period s is a non-configurable 7 days - do we want to be specific here, or are we worried that it could change in the future?

Reply:
Changed!